### PR TITLE
WIP Update maintainers.json to list mcmillhj as alumnus

### PR DESCRIPTION
**Pull request by mcmillhj**
Based on this conversation, https://github.com/exercism/sml/issues/97#issuecomment-420808797, I am updating the `config/maintainers.json` file to list myself as an alumnus.

### CODE REVIEW

File: config/maintainers.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     },
     {
       "github_username": "mcmillhj",
-      "alumnus": false,
+      "alumnus": "alumnus",
```
Comment (snahor) [inline config/maintainers.json]: Hey @mcmillhj I think the value should be `true` instead of `alumnus`, do you want me to fix it?

Comment (mcmillhj) [reply config/maintainers.json]: @snahor I thought the same thing at first, but from the docs it looks like the only accepted values are `alumnum`, `alumnus` or `alumna`: https://github.com/exercism/docs/blob/master/maintaining-a-track/maintainer-configuration.md

Comment (snahor) [reply config/maintainers.json]: Hmm, I guess that document is outdated. I've checked other repos and they are using true/false.

Comment (mcmillhj) [reply config/maintainers.json]: hmm okay, I'll update to a boolean one second.

**Comment by mcmillhj** [comment]
I borked this somehow, I may open a new PR since that seems easier.

**Pull request closed without merging**